src/rtcmix/Makefile

```diff
@@ -214,7 +214,6 @@ ifeq ($(RTLIBTYPE), STATIC)
 		$(LIBPFBUSOBJS) $(LIBMINCOBJS) $(LIBOBJOBJS) $(LIBRTHEAPOBJS) \
 		$(LIBSNDOBJS) $(LIBSTKOBJS)
 	LOAD = $(AR) rv
-	RANLIB = ranlib
```

Comment:
Did you verify that all other configurations (in makefile.conf.in) provide a macro value for RANLIB?

Reply:
I didn't at the time, but it looks like all of the architectures (Linux, macOS, SGI :upside_down_face:, FreeBSD) do provide a value for `RANLIB`.